homeassistant/components/command_line/sensor.py

```diff
@@ -171,7 +170,7 @@ def update(self):
             pass
         else:
             # Template used. Construct the string used in the shell
-            command = str(" ".join([prog] + shlex.split(rendered_args)))
+            command = f"{prog} {rendered_args}"
```

Comment:
I think this could be a bit bad. I think f strings allow nested arguments. So if `rendered_args` or `prog` just so happen to contain {hass.some_secret_variable}. We could expose internal variables here.

Reply:
Okey. it doesn't seem to expand inside an expanded variable. Puh.. So i retract my previous comment